changelog/@unreleased/pr-4707.v2.yml

```diff
@@ -0,0 +1,8 @@
+type: improvement
```

Comment:
We could mark this as `break` if we want to make the ABI break more obvious.

Reply:
Good call, version numbers are cheap :-)